save-cloud-common/src/jvmMain/kotlin/org/cqfn/save/entities/LnkUserProject.kt

```diff
@@ -0,0 +1,16 @@
+package org.cqfn.save.entities
+
+import org.cqfn.save.user.UserRole
+import javax.persistence.Entity
+
+/**
+ * @property project
+ * @property user in project
+ * @property role role of this user
+ */
+@Entity
+class LnkUserProject(
+    var project: Project?,
+    var user: User?,
+    var role: UserRole?,
```

Comment:
We should also remove columns `user_id` and `admin_ids` from `project` and use data from this mapping

Reply:
Yes. I will do this in another PR because it will involve a lot of code refactoring. Here I wanted to do only adding new tables and classes

Reply:
Ok, no problem